src/model/encoding/__tests__/convertFromRawToDraftState-test.js

```diff
@@ -16,19 +16,110 @@ jest.disableAutomock();
 
 const convertFromRawToDraftState = require('convertFromRawToDraftState');
 
-describe('convertFromRawToDraftState', () => {
-  it('must map falsey block types to default value of unstyled', () => {
-    const rawState = {
-      blocks: [
-        {text: 'AAAA'},
-        {text: 'BBBB', type: null},
-        {text: 'CCCC', type: undefined},
-      ],
-      entityMap: {},
-    };
-
-    const contentState = convertFromRawToDraftState(rawState);
-    const blockMap = contentState.getBlockMap();
-    blockMap.forEach(block => expect(block.type).toEqual('unstyled'));
-  });
+test('must map falsey block types to default value of unstyled', () => {
+  const rawState = {
+    blocks: [
+      {key: 'A', text: 'AAAA'},
+      {key: 'B', text: 'BBBB', type: null},
+      {key: 'C', text: 'CCCC', type: undefined},
+    ],
+    entityMap: {},
+  };
+
+  expect(convertFromRawToDraftState(rawState).getBlockMap()).toMatchSnapshot();
+});
+
+test('must be able to convert from styled blocks and entities mapped raw state', () => {
```

Comment:
Added this extra test, since we did not had any tests that checked that styles were created properly